plugin/webhook/custom.go

```diff
@@ -16,16 +16,16 @@ type CustomWebhookResponse struct {
 
 // CustomWebhookRequest is the API message for Custom webhook request.
 type CustomWebhookRequest struct {
-	Level        Level   `json:"level"`
-	ActivityType string  `json:"activity_type"`
-	Title        string  `json:"title"`
-	Description  string  `json:"description"`
-	Link         string  `json:"link"`
-	CreatorID    int     `json:"creator_id"`
-	CreatorName  string  `json:"creator_name"`
-	CreatedTS    int64   `json:"created_ts"`
-	Issue        Issue   `json:"issue"`
-	Project      Project `json:"project"`
```

Comment:
If fields are not designed to be possibly empty, we should avoid using pointers.

Reply:
But it may be empty, for example:https://sourcegraph.com/github.com/bytebase/bytebase/-/blob/server/project_webhook.go?L201. So I think we need a way to determine if an `issue` is included